server/pxf-api/src/main/java/org/greenplum/pxf/api/io/DataType.java

```diff
@@ -158,4 +176,6 @@ public int getOID() {
     public DataType getTypeElem() {
         return typeElem;
     }
+
+    public DataType getTypeArray() { return typeArray; }
```

Comment:
can we match styling as above?
```suggestion
    public DataType getTypeArray() { 
        return typeArray; 
    }